runpod/serverless/worker.py

```diff
@@ -74,20 +76,25 @@ async def process_job(job):
 
                 # If refresh_worker is set, pod will be reset after job is complete.
                 if config.get("refresh_worker", False):
-                    log.info(f"refresh_worker | Flag set, stopping pod after job {job['id']}.")
+                    log.info(
```

Comment:
Is your text editor wrapping this line? I belive it should be less than 100 as is.